### PR TITLE
Add maximum/minimum attributes for numbers

### PR DESCRIPTION
**Pull request by ambs**
While enums aren't supported, maximum and minimum will help me.
Unfortunately the exclusiveMaximum and minimum are not that easy to solve for non-Integers.
Suggestions are welcome.

### CODE REVIEW

File: lib/JSON/Schema/Fit.pm [hunk 1]
```diff
@@ -86,6 +86,19 @@ Default: 1
 
 sub round_numbers { return _attr('round_numbers', @_); }
 
+=attr limits
```
Comment (liosha) [inline lib/JSON/Schema/Fit.pm]: Something like 'crop_numbers' will be better

Comment (ambs) [reply lib/JSON/Schema/Fit.pm]: Hi. Curiously, I replied to github email, but I do not see here the thread. 
Some libraries use clamp for this kind of operation: https://docs.unity3d.com/ScriptReference/Mathf.Clamp.html
What do you prefer, `crop` or `clamp`?

**Comment by ambs** [comment]
Given the lack of answer, gone with crop. Still happy to change to `clamp`.

File: lib/JSON/Schema/Fit.pm (hunk 1 above)
Comment (liosha) [reply lib/JSON/Schema/Fit.pm]: Hi! Sorry for the delay.
Clamp is better, i just hadn't found this term.
I'll fix it myself

**Comment by ambs** [comment]
No problem. Thanks!

**Pull request merged**